Doc/whatsnew/3.14.rst

```diff
@@ -97,6 +97,7 @@ Added :func:`ast.compare` for comparing two ASTs.
 Optimizations
 =============
 
+* The default protocol in the :mod:`pickle` module is now Protocol 5.
```

Comment:
I don't personally have context to add here. Maybe another reviewer does.

Reply:
Maybe "Set the default protocol version of the <code>:mod:\`pickle\`</code> module to ``5``"? Or mention the corresponding constant?

Reply:
Mmh. I think it's perhaps good to simply add something like "See :pep:\`3118\` for details." or soemthing along the lines or anything related to what Protocol 5 brings (if it's a section in the pickle page, you can refer to it maybe?)

Reply:
oh, cool, will look to something like that (or use your suggestion as fallback)

Reply:
I've mentioned pickle's "data stream format" section, it seems to be a very good source of detailed information about it :)